docs/images/siddhi-logo-w.svg

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Change the year. Please change at the other places as well

Reply:
Fixed by 71dad9c6b3c676a5a94d62bcb9ce7fcd5b91e918